### PR TITLE
Spacy helper

### PR DESCRIPTION
**Pull request by isNeil**

### CODE REVIEW

File: examples/spacy/spacy-tokenizer.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+    @pipeline_function
+    def predict(self, input: str) -> list:
+        doc = self.nlp(input)
+        # (optional) your spacy code here or you can return entire spacy object
+        # to manipulate on a client that has spacy installed.
+        res = []
+        for token in doc:
+            res.append([token.text, token.pos_, token.dep_])
+        return res
```
Comment (rossgray) [inline examples/spacy/spacy-tokenizer.py:23]: just a style thing so not important, but I quite like using list comprehensions for this kind of thing, e.g.
```python
return [[token.text, token.pos_, token.dep_] for token in doc]

Comment (alexpearce) [inline examples/spacy/spacy-tokenizer.py:23]:
```suggestion
        return [[token.text, token.pos_, token.dep_] for token in doc]
```

**Review by alexpearce — COMMENTED** [review]
Just one suggestion for your consideration, otherwise LGTM!

I guess we don't need to bump the version when adding examples but no harm either.

**Review by rossgray — APPROVED** [review]
Looks good to me

**Review by alexpearce — APPROVED** [review]
🚢

**Pull request merged**